regolith/helpers/u_milestonehelper.py

```diff
@@ -16,10 +18,10 @@ def subparser(subpi):
     subpi.add_argument("projectum_id", help="The id of the projectum.")
     subpi.add_argument("-v", "--verbose", action="store_true",
                         help="Increases the verbosity of the output.")
-    subpi.add_argument("-i", "--index", nargs='+',
+    subpi.add_argument("-i", "--index",
                         help="Index of the item in the enumerated list to update. "
-                             "If multiple indexes, separate by space",
-                        type = int)
+                             "Please enter in the format of `2,5,7` or `3-7`",
```

Comment:
this isn't quite clear.  do you need left-leaning quotes or not?  No spaces in the list?  I guess you are going to parse as as string?  Can I leave a single integer or do I have to do `2` for just closing one milestone?  Preferrable would be if it is an int it will do just one in that case